lib/omnibus/s3_cache.rb

```diff
@@ -129,33 +128,13 @@ def key_for(software)
 
       private
 
-      #
-      # The client to connect to S3 with.
-      #
-      # @return [UberS3::Client]
-      #
-      def client
-        @client ||= UberS3.new(
-          access_key:        Config.s3_access_key,
-          secret_access_key: Config.s3_secret_key,
-          bucket:            Config.s3_bucket,
-          adapter:           :net_http,
-        )
-      end
-
-      #
-      # The bucket where the objects live.
-      #
-      # @return [UberS3::Bucket]
-      #
-      def bucket
-        @bucket ||= begin
-          if client.exists?('/')
-            client.bucket
-          else
-            client.connection.put('/')
-          end
-        end
+      def s3_configuration
+        {
+          region:               'us-east-1',
```

Comment:
we can make it configurable. IIRC, us-east-1 will forward to the correct region, so we can use that as a fallback.